Jmg047/CSC648-demo-team03
# feat(api/api-server.js):

Created an API to fetch all gigs and workers as well as custom query for the search bar.

**to test:**
Open terminal, cd to application folder
run a `git fetch` and a `git switch feat-demo-backend`
make sure to run `yarn` to download any and all dependencies
run `yarn build` to make sure the project compiles
[Optional] have `yarn start` and `yarn watch` going in separate terminals, you should see a "This is a demonstration via React Frontend" render on the frontend.
in a separate terminal, cd to the application folder and run `node api/api-server.js`

**In postman:**
- try out a GET request on: 'http://localhost:2000/workers/652619af75d0d9b0acb7218e?FirstName=aLICE'
should return a worker named alice johnson.
ry out a GET request on: 'http://localhost:2000/get-gigs/?title=assistant'
should return a gig titled: 'Assist with general house cleaning and organizing tasks.'
- use different keys like `FirstName', LastName', or 'Location' and try out different input combinations. [Worker]
- - use different keys like `title', 'location', and try out different input combinations. [Gigs]
- all the JSON data for gigs will be at: http://localhost:2000/gigs
- all JSON data for workers will be at: http://localhost:2000/workers
